opennlp-tools/src/main/java/opennlp/tools/ml/maxent/RealBasicEventStream.java

```diff
@@ -49,13 +55,14 @@ public Event read() throws IOException {
   }
 
   private Event createEvent(String obs) {
-    int lastSpace = obs.lastIndexOf(' ');
-    if (lastSpace == -1)
+    int si = obs.indexOf(' ');
+    if (si == 0)
```

Comment:
Shouldn't that be `-1` ? Otherwise, it might produce an index out of bound exception in the else branch.